src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let amount = 0;
+
+  const adder = (arg) => {
+    if (arg !== undefined) {
```

Comment:
Consider using `typeof arg !== 'undefined'` for a more explicit check against `undefined`. This prevents potential bugs when falsy values are passed to `adder`.